toolchain/lower/file_context.cpp

```diff
@@ -360,6 +362,18 @@ auto FileContext::BuildFunctionDefinition(SemIR::FunctionId function_id)
   }
 }
 
+auto FileContext::BuildSubprogram(
+    const Carbon::SemIR::Function& /*semir_function*/,
```

Comment:
```suggestion
    const Carbon::SemIR::Function& /*function*/,
```

Match the header? (clang-tidy might catch this, if you're watching in vscode)

Reply:
Done